src/module/actor/data-model-classes/__tests__/data-model-character-move.test.ts

```diff
@@ -147,27 +147,97 @@ export default ({
       expect(move.encounter).to.equal(expectedEncounter)
       expect(move.overland).to.equal(expectedOverland)
     })
-    it('At 50% encumbered', () => {
+    it('At 25.1% encumbered (75% moverate)', () => {
+      const expectedBase = OseDataModelCharacterMove.baseMoveRate * .75;
```

Comment:
It would be cool if we could DRY up these tests somehow.

Reply:
yeaaah